rest-api/data/sites.csv

```diff
@@ -1,54 +1,54 @@
+AZ_TUCSON_BANNER_HEALTH,hpo-site-bannerboswell,Banner Boswell Medical Center,HPO Banner Boswell,7036426,,,Active,Active,Active,Active,Inactive,Inactive,Active,Inactive,Inactive,Active,Inactive,Inactive,,,,10401 W. Thunderbird Rd.,,Sun City,AZ,85351,,owilliams@email.arizona.edu,
+AZ_TUCSON_BANNER_HEALTH,hpo-site-banneralvernonclinic,Banner University Medical Center Alvernon Clinic,HPO Banner Alvernon Clinic,7036647,4/30/18,,Active,Active,Active,Active,Inactive,Inactive,Active,Inactive,Inactive,Active,Inactive,Inactive,,,,655 N Alvernon Way,,Tucson,AZ,85716,,"mwences@email.arizona.edu, jillianwilson@email.arizona.edu",
+AZ_TUCSON_BANNER_HEALTH,hpo-site-bannernorthclinic,Banner University Medical Center North Clinic,HPO Banner North Clinic,7036646,1/5/18,,Active,Active,Active,Active,Inactive,Inactive,Active,Inactive,Inactive,Active,Inactive,Inactive,,,BUMC North Clinic,3838 N Campbell Ave,,Tucson,AZ,85719,,"mwences@email.arizona.edu, jillianwilson@email.arizona.edu",
+AZ_TUCSON_BANNER_HEALTH,hpo-site-bannergateway,Banner Gateway Medical Center,HPO Banner Gateway,7036693,4/23/18,,Active,Active,Active,Active,Inactive,Inactive,Active,Inactive,Inactive,Active,Inactive,Inactive,,,,1900 N. Higley Way,,Gilbert,AZ,85234,,dianaabril@email.arizona.edu,
+AZ_TUCSON_BANNER_HEALTH,hpo-site-bannerbaywood,Banner Baywood Medical Center,HPO Banner Baywood,7036694,4/23/18,,Active,Active,Active,Active,Inactive,Inactive,Active,Inactive,Inactive,Active,Inactive,Inactive,,,,6644 E. Baywood Ave.,,Mesa,AZ,85206,,dianaabril@email.arizona.edu,
+AZ_TUCSON_BANNER_HEALTH,hpo-site-bannerdelewebb,Banner Del E Webb Medical Center,HPO Banner Del E Webb,7036695,5/7/18,,Inactive,Active,Active,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,,,,14502 W. Meeker Blvd,,Sun City West,AZ,85375,,,
+AZ_TUCSON_BANNER_HEALTH,hpo-site-bannermariposachc,Banner Mariposa Community Health Center,HPO Banner Mariposa  CHC,7036696,5/7/18,,Inactive,Active,Active,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,Inactive,,,,1852 N. Mastick Way ,,Nogales,AZ,85621,,,
+CAL_PMC_SDBB,hpo-site-sandiegobb,HPO San Diego Blood Bank (SDBB) ,HPO SDBB,7035957,,,Active,Active,Active,Active,Inactive,Inactive,Active,Inactive,Inactive,Active,Inactive,Inactive,,,,3637 Gateway Center Ave,,San Diego,CA ,92102,,"rkurtz@ucsd.edu, jvilla@sandiegobloodbank.org",
+CAL_PMC_SDBB,hpo-site-sandiegonorthcounty,HPO San Diego Blood Bank North County Donor Center (NCDC),HPO SDBB North County,7036687,,,Active,Active,Active,Active,Inactive,Inactive,Active,Inactive,Inactive,Active,Inactive,Inactive,,,,"358 West El Norte Parkway 
 ",Suite J ,Escondido,CA ,92026,,"jvilla@sandiegobloodbank.org, jarandela@sandiegobloodbank.org",

```

Comment:
I hadn't noticed this before, but: we should make sure that our CSV parsing handles newlines in the middle of string fields like this appropriately. This might be the cause of the warnings in the logs.

Reply:
(If we aren't handling this appropriately, we may need to specify the newline parameter to our CSV reader...)